### DISCUSSION TITLE
Community Translations

### DISCUSSION THREAD

**Discussion by SebLague** [SebLague/Geographical-Adventures#70] [Translation]
Hi everyone, thanks for all the interest in translating the game into different languages! (#68)
I have begun the process of adding support for this into the game, and at the moment the following files are ready to be translated:
* MenuLocalization.json
* CountryLocalization.json
* Manual.txt

These files can be found and edited here: https://github.com/SebLague/Geographical-Adventures/tree/main/Assets/Data/Localization
The main things missing from the localization support at the moment are: city names, in-game messages, the controls UI, and the statistics screen. I will work on adding these in the future.

Since this game is still under construction, a lot of text might still be added/changed/removed, so please take that into consideration when deciding whether you want to invest time into making translations at this stage.

Once you have made some translations you can submit a pull request and I will import them into the game.

Please post here if you have started/stopped working on a particular language to avoid others doing duplicate work.
You can also leave a message here if you'd like to work on a language that has not yet been added, and I will set it up.

**Comment by fsioni** [comment]
I'm in for the french translation !
Edit : PR created !

**Comment by tposejank** [comment]
I'm gonna work on the Spanish translation in a few hours.

**Reply by 09dg** [reply]
I have just collaborated with the translation of all countries in Spanish.

**Comment by Agentew04** [comment]
Just created a PR for the Portuguese translation too

**Reply by 09dg** [reply]
I'm there translating some stuff and making small corrections.

**Comment by MillersMan** [comment]
I'm gonna work on the German translation in a few hours

**Reply by Racooder** [reply]
I added some language and spelling enhancements.

**Comment by fsioni** [comment]
With the new rebinding menu, it could be great to have it translated as well :)

**Comment by StuSerious** [comment]
Working on the Italian i18n right now!

**Reply by StuSerious** [reply]
Merged in https://github.com/SebLague/Geographical-Adventures/pull/76!

**Comment by SebLague** [comment]
I've updated the MenuLocalization.json file to include entries for the control bindings. The localized country names are also now being used in the game. Thanks for all the translations everyone!

**Reply by StuSerious** [reply]
I'll get to translating in IT later today or tomorrow!

**Comment by tposejank** [comment]
The spanish translation is almost finished.

**Comment by tposejank** [comment]
#82 Translation pr created

**Comment by MillersMan** [comment]
#83 German translation of countries created.
Thanks to @EvilsSouls for translating the menu and manual as I was struggling to keep those up to date

**Comment by SebLague** [comment]
Thanks everyone -- version 1.0.6 is now out on [itch.io](https://sebastian.itch.io/geographical-adventures) and includes all the translations done so far. I've added your usernames to the credits, but let me know if you'd prefer to have your real name displayed (or if you'd prefer not to be listed at all).

**Comment by IAMACAR10** [comment]
I can look into doing swedish translation :)

**Reply by ghost** [reply]
Great

**Comment by maticjesenicnik** [comment]
Started the work on the **Slovenian** translation.

**Reply by maticjesenicnik** [reply]
Finished the translations and opened a PR.

**Comment by IAMACAR10** [comment]
Finished all the countries on the Swedish translation, still have menus & manual to go

**Reply by ghost** [reply]
Awesome

**Comment by IAMACAR10** [comment]
Hey, sorry it took so long. I am almost finished, will open a PR soon

**Reply by ghost** [reply]
Thx

**Reply by ghost** [reply]
I am from Sweden so it’s nice to see someone translating this great game into my home language!

**Reply by ghost** [reply]
When will you publish it?

**Reply by ghost** [reply]
Add the goddamn pr!

**Reply by SebLague** [reply]
Please be polite @h57758, Car has no obligation to finish this.

**Reply by ghost** [reply]
> Please be polite @h57758, Car has no obligation to finish this.

Sorry, but the thing is that they already have the fork ready but not the pr

**Reply by Maltavius** [reply]
@IAMACAR10 So are you done with it or should someone start over?

**Comment by Triogap** [comment]
Hello everybody. Lovely project and just added a PR for Dutch translations(#100).
